apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryJob.cs

```diff
@@ -107,17 +107,19 @@ public BigQueryJob ThrowOnAnyError()
             var errors = Resource.Status?.Errors;
             if (errors?.Count > 0)
             {
-                throw new GoogleApiException(_client.Service.Name, $"Job {Reference?.ProjectId}/{Reference?.Location}/{Reference?.JobId} contained errors")
+                throw new GoogleApiException(_client.Service.Name)
                 {
                     Error = new RequestError
                     {
                         Errors = errors.Select(error => new SingleError
-                            {
-                                Location = error.Location,
-                                Reason = error.Reason,
-                                Message = error.Message
-                            })
-                            .ToList()
+                        {
+                            Location = error.Location,
+                            Reason = error.Reason,
+                            Message = error.Message
+                        }).ToList(),
+                        Message = errors.Count == 1
+                            ? errors.First().Message
```

Comment:
If there's exactly one error, do we still want the job reference to be easily available? (Just trying to make sure we don't make information harder to use by trying to simplify things.)

Reply:
Yes, I'll reformat this message to include the job reference as well.